python/ray/tune/tests/test_trial_scheduler_pbt.py

```diff
@@ -122,6 +126,10 @@ def load_checkpoint(self, path):
                 with open(path, "rb") as fp:
                     self.iter, self.a = pickle.load(fp)
 
+            def reset_config(self, new_config):
+                self.a = new_config["a"]
+                return True
+
```

Comment:
See above

Reply:
pbt uses `reset_trial` directly instead of going through `reuse_actors`.